content/docs/09.5-security/10-hardened-infrastructure.md

```diff
@@ -0,0 +1,153 @@
+
+    
+    - User account name and email ID.
+    
+    
+    - Tenant's cloud credentials.
+
+    
+    - Tenant's private registry password.
+
+    
+    - Tenant's GitHub access token.
+
+
+In self-managed deployments, keys are generated during installation and stored as secrets in the management cluster’s etcd key-value store.
+
+
+<br />
+
+## Data Encryption
+
+In SaaS deployments, backup snapshots of MongoDB cluster storage volumes are encrypted to secure cluster data on disk. In tenant clusters, Palette ensures security for data at rest and data in transit. Refer to [Tenant Cluster Security](/security/saas-operation#tenantclustersecurity) to learn how Palette applies data security.
+
+In self-hosted deployments, the administrator can set up a disk encryption policy for management cluster VMs if required. Refer to [Self-Hosted Operation](/security/self-hosted-operation) for details.
+
+<br />
+
+## Secure Communications
+
+Palette secures data in transit using an encrypted Transport Layer Security (TLS) communication channel for all internal and external interactions. Review [Tenant Cluster Security](/security/saas-operation#tenantclustersecurity) to learn how Palette applies security to data in transit.
+
+Public certificates are created using a cert-manager for external API/UI communication. For on-prem deployment, you can import an optional certificate and private key to match the management cluster Fully Qualified Domain Name (FQDN).
+
+Services in the management cluster communicate over HTTPS with self-signed certificates and Rivest–Shamir–Adleman (RSA) 2048-bit key.
+
+The database connection from application services running in the management cluster to MongoDB is protected by TLS with Authentication enabled. 
+
+Static Network Address Translation (NATS) messages are exchanged using TLS protocol, and each tenant cluster uses dedicated credentials to connect to the message bus. Authentication and Authorization policies are enforced in the NATS deployment to ensure message and data isolation across tenants. 
+
+<br />
+
+## Authentication and Authorization
+
+Palette fully supports Role-Based Access Control (RBAC) and two authentication modes:
+
+<br />
+
+- *Local authentication* and *password policy* <br />
+
+    With local authentication, a user email serves as the ID, and a password is compared with the one-way hash stored in the database to authenticate users to a tenant. The platform administrator can set password policy to control the requirements for password length, rule, and expiration.
+
+
+- *Single Sign-On (SSO)* and *Multi-Factor Authentication (MFA)* <br />
+
+    In these modes, the tenant is configured to have Security Assertion Markup Language (SAML) 2.0 IDP integrations. If the IDP requires MFA, you are redirected to the IDP’s authentication page. SSO can also map a user to one or more user groups in the tenant automatically.
+
+
+## API Security
```

Comment:
15 min is what I took from the API doc. @ritawatson changed it to "limited time" as it will be customizable soon with a default timeout that will be higher than 15m.